corehq/apps/hqcase/api/updates.py

```diff
@@ -275,3 +300,84 @@ def _submit_case_blocks(case_blocks, domain, user, device_id):
         device_id=device_id,
         max_wait=15
     )
+
+
+def _get_owner_ids(data, case_db):
```

Comment:
I think something like `_get_owner_and_case_ids_from_data` will make it a bit clearer what this function does

Reply:
I've moved the logic for converting case to owner id into the helper function. Only owner_ids will be returned, so the `_get_owner_ids()` function name is still relevant. :+1: